mmv1/third_party/terraform/services/pubsub/resource_pubsub_subscription_test.go

```diff
@@ -238,6 +238,10 @@ func TestAccPubsubSubscriptionBigQuery_serviceAccount(t *testing.T) {
 	topic := fmt.Sprintf("tf-test-topic-%s", acctest.RandString(t, 10))
 	subscriptionShort := fmt.Sprintf("tf-test-sub-%s", acctest.RandString(t, 10))
 
+	if acctest.BootstrapPSARoles(t, "service-", "gcp-sa-pubsub", []string{"roles/bigquery.dataEditor", "roles/bigquery.metadataViewer"}) {
```

Comment:
@melinath or @NickElliot , do you know the reason to stop the test because roles were added to IAM policy in other examples?

Reply:
I've been wondering about this a little too, because in my limited testing, the bindings seem to get removed again when the test is over? But people generally talk about this only needing to fail on the first run, so maybe it is "sticky" (or supposed to be)?

From the comments, it seems to be because the binding may not show up quickly enough and the test might fail downstream? I couldn't really find any examples where it didn't do this.

Reply:
Also, does it need to add the role when `VCR_MODE=REPLAYING` is set also? I got an exit locally running this with `VCR_MODE=REPLAYING`

Reply:
I guess the second run will success, as the role has been added in the first time.